src/index.js

```diff
@@ -1,92 +1,44 @@
 /* eslint-disable no-labels */
 
-const FS = require('fs');
-const Path = require('path');
-const EventEmitter = require('events');
-
-const { PNG } = require('pngjs');
-const JPEG = require('jpeg-js');
-const BMP = require('bmp-js');
-const GIF = require('omggif');
-const UTIF = require('utif');
-const MIME = require('mime');
-const tinyColor = require('tinycolor2');
-const BigNumber = require('bignumber.js');
```

Comment:
I'm not in favor of using import syntax as it is not yet supported by node.

I would like to keep node-compatibility for the modules without any transpilation.

Reply:
we already babel so why not transpile for node as well?

Reply:
it's literally as easy as pointing to `dist/index.js` rather than `src/index.js`

Reply:
With this change we can also start supporting the `module` field in the package.json

https://github.com/nodejs/node-eps/blob/4217dca299d89c8c18ac44c878b5fe9581974ef3/002-es6-modules.md#51-determining-if-source-is-an-es-module

Reply:
a tree-shakable jimp is a goal of mine.

Reply:
https://github.com/webpack/webpack/issues/4674

Reply:
we already use a bunch of other es6 features as well that we should be transpiling for older platforms (const, let). why not import/export?

Reply:
After a bit of further investigation it seems like tree-shaking/elimination of unused class/prototype methods isn't really possible.

I'm still in favor of using import/export but am interested in reason why not to? We use es6 features not supported for node <6.4.0 so transpiling should already be happening.

Reply:
Although if they were to use the closure compiler unused prototype methods seem to be eliminated.

https://closure-compiler.appspot.com/home#code%3Dclass%2520Foo%2520extends%2520EventEmitter%2520%257B%250A%2509toString()%2520%257B%250A%2509%2509return%2520'fooooo'%253B%250A%2520%2520%2520%2520%257D%250A%257D%250A%250Aconst%2520toBeEliminated%2520%253D%2520()%2520%253D%253E%2520'be%2520gone!'%253B%250AFoo.prototype.toBeEliminated%2520%253D%2520toBeEliminated%250A%250Aconsole.log('here')%250Aconst%2520foo%2520%253D%2520new%2520Foo()%253B%250Aconsole.log(Foo.toString())